cilium/cmd/encrypt_status.go

```diff
@@ -37,6 +59,43 @@ func init() {
 	command.AddJSONOutput(encryptStatusCmd)
 }
 
+func getIpsecmode() string {
```

Comment:
I think that `getEncryptionMode` would be a better name since the function may also return the encryption mode for WireGuard.